app/models/expense.rb

```diff
@@ -0,0 +1,8 @@
+class Expense < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Kindly change this association to match the ERD guidelines, a category should belong to author like so `belongs_to :author, class_name: User`. This change should be reflected in the scheme. Categories table should have an `author_id` instead of `user_id`.